tasks/skylab/Metrics.wdl

```diff
@@ -220,7 +242,7 @@ task FastqMetricsSlideSeq {
 
     # Runtime attributes
     #String docker =  "us.gcr.io/broad-gotc-prod/warp-tools:1.0.9-1700252065"
-    String docker = "us.gcr.io/broad-gotc-prod/warp-tools:2.0.0"
+    String docker = "us.gcr.io/broad-gotc-prod/warp-tools:aa-mt-tagsort"
```

Comment:
should this be updated to us.gcr.io/broad-gotc-prod/warp-tools:2.0.1 ?

Reply:
Just updated it